src/testing/rapid_tests.py

```diff
@@ -325,3 +372,21 @@ def _randomize_rapid_tests(states, target_share_to_be_tested, share_refuser, see
     rapid_test_demand = pd.Series(False, index=states.index)
     rapid_test_demand[to_test_indices] = True
     return rapid_test_demand
+
+
+def _only_not_fully_vaccinated_test_themselves(rapid_test_demand, states):
+    """Exclude fully vaccinated individuals from being tested.
+
+    The immunity countdown is initialized at -9999. The vaccine countdown is set
+    between -1 and 21. This is only the 1st vaccine. Assuming 30 days between
+    shots and 14 days of wait period after the 2nd shot, individuals are freed
+    from testing obligations ~45 days after their first shot. This translates
+    into countdown values between -45 and -20. For simplicity we simply assume
+    that individuals stop testing themselves 40 days after their first shot.
```

Comment:
Wouldn't this change with Tim's sid PRs? In any case, it seems to me like we should not rely too much on sid implementation details.